OpenRA.Mods.Common/Traits/Turreted.cs

```diff
@@ -117,7 +117,7 @@ void ITick.Tick(Actor self)
 		protected virtual void Tick(Actor self)
 		{
 			// NOTE: FaceTarget is called in AttackTurreted.CanAttack if the turret has a target.
-			if (attack != null)
+			if (!IsTraitDisabled && attack != null)
```

Comment:
I don't think this is quite right either: the disabled case will now hit the `else` branch and `MoveTurret()`!
Is there a reason you've been avoiding adding `if (IsTraitDisabled) \n return;` as its own check at the start?